docker/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
 #    image: percona:5.7
```

Comment:
Right now last 5.7 version has some issue causing the container to go into a restart loop so we are using 5.7.22 specific version. Mb we should make this change here or new PR?

Reply:
Better wait for new release and file issue upstream

Reply:
5.7.24 is out with secfixes